### ISSUE TITLE
Package is broken because https://qaos.math.tu-berlin.de/qaos is offline

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/qaos#13]
Either https://qaos.math.tu-berlin.de/qaos needs to be resurrected (possibly at a new address), or else this package is completely broken, and thus should be withdrawn.

**Comment by olexandr-konovalov** [comment]
Hm. This page works http://page.math.tu-berlin.de/~kant/database.html, but crucial links there are broken.

**Comment by fingolfin** [comment]
That page is not the QaoS API, so it's not relevant.

**Comment by olexandr-konovalov** [comment]
I see that. Maybe @hroptatyr / @freundt knows?

**Comment by hroptatyr** [comment]
Hi Alex,
yes, I can confirm that the machine that hosted the qaos database is decommissioned.

**Comment by fingolfin** [comment]
@hroptatyr Thanks for the info. So will it stay decommissioned? I.e., this package is permanently broken?

@alex-konovalov then we should remove this package from the distribution. What steps have to be taken?

**Comment by olexandr-konovalov** [comment]
Same like https://mail.gap-system.org/pipermail/gap/2018-March/001082.html and https://mail.gap-system.org/pipermail/gap/2018-September/001128.html (email to that mailing list explaining this). This repository will stay here, and for the package updates system I will archive the mercurial repository with release snapshots, and will remove the URL of PackageInfo.g from the registry.

**Comment by hroptatyr** [comment]
@fingolfin Yes, it's been decommissioned for good. Moreover, with Pohst's retirement maintenance of the data (and/or the query system (qaos)) has stopped.

**Comment by olexandr-konovalov** [comment]
@hroptatyr Thanks. So what happens next? Is there anyone who is keeping the data and is willing to resurrect this elsewhere, or at least conserve them in some repository?

**Comment by olexandr-konovalov** [comment]
As commit https://github.com/gap-system/gap-distribution/commit/337de02e70a2f49ecfa76a9f6208d0294bff189a says:

Unfortunately we had to withdraw the `QaoS` package from distribution of GAP, as the servers it crucially relies on for its functionality have been permanently retired some time ago and are not coming back (see <https://github.com/gap-packages/qaos/issues/13> for details).  Without them, this package is mostly useless.

Of course the source code repository on GitHub will stay around, just in case somebody is interested in the code for other reasons.
